src/NCronJob/Scheduler/QueueWorker.cs

```diff
@@ -176,7 +176,7 @@ private void AddWorkerTask(string jobQueueName, CancellationToken stopToken)
 
     private void ScheduleInitialJobs()
     {
-        foreach (var job in jobRegistry.GetAllCronJobs())
+        foreach (var job in jobRegistry.GetAllCronJobs().Where(j => j.IsEnabled))
```

Comment:
More a scratch of a personal itch than a true internal optimization.

I usually disable jobs when running integration tests. For some reason, I had to step debug the startup and eventually hit that useless looping.

Reply:
Fair and probably something we want to goto anyway